source/new-ia/by-role/system-admin/authoring/configure-apache-vhost-authoring.rst

```diff
@@ -0,0 +1,42 @@
+:is-up-to-date: True
+:since-version: 4.0.3
+:nosearch:
+
+.. index:: Configuring Apache Virtual Host for Authoring, Configuring Apache vhost
+
+.. _newIa-configure-apache-vhost-for-authoring:
+
+=============================================
+Configuring Apache Virtual Host for Authoring
+=============================================
+
+.. version_tag::
+   :label: Since
+   :version: 4.0.3
+
+This section will show you a sample Apache virtual host configuration for your CrafterCMS project in authoring.
```

Comment:
show you an example reverse proxy using Apache 2 HTTPd vhost configuration for Crafter Studio.